XROLE/curtainAndBlind
# Ch setup travis #166462699

*What does this PR do?*
- This PR integrate Travis CI into CurtainAndBlind

*Description of task to be completed*
- [x] add and configure .travis.yml file

*How should this be manually tested?*
- navigate to [travis](https://travis-ci.org/XROLE/curtainAndBlind/builds/541588124) to see the build passing

*What are the relevant pivotal tracker stories?*
[#166462699](https://www.pivotaltracker.com/story/show/166462699)
